packages/skin/src/routes/_index/component/selection-chip/+page.marko

```diff
@@ -0,0 +1,43 @@
+<div id="chip">
+    <section-header metadata=metadata/>
+
+    <h3 id="selection-chip">
+        Selection Chip
+    </h3>
+    <p>An selection chip toggles between two states, selected and unselected.</p>
+    <p>A selection chip is used in a form as a checkbox. This is generally used to select predetermined values</p>
```

Comment:
This doesn't seem right. It's not really a checkbox. May be best to just delete this line. I can actually sync up these parts with the MIND pattern docs if you like.

Reply:
Sure. I haven't updated the text for this yet but if you want to I'm fine with that

Reply:
Done